docs/InformedConsent/InformedConsent-template-Swfit3.md

```diff
@@ -0,0 +1,206 @@
+section1.title = "The title of the section goes here ..."
+section1.summary = "The summary about the section goes here ..."
+section1.content = "The content to show in learn more ..."
+
+// Create additional section objects for later sections
+document.sections = [section1, ...]
+
+let step = ORKVisualConsentStep(identifier: "VisualConsentStep", document: document)
+// And then create and present a task including this step.
+```
+
+If the predefined consent sections do not adequately cover the sections of your consent document, you can create your own custom sections. You can also create your own images and animations and add them to your consent section, to complete the experience. The animations you add should be H.264 videos; for best results, try to match the assets included with the ResearchKit framework.
+
+### Create a Visual Consent Step
+
+The visual consent step presents the sections of the consent document.
+It is your responsibility to populate the visual consent step with
+content; the `ORKVisualConsentStep` object doesn't contain any
+default content.
+
+```swift
+// Add consent sections for each page of visual consent; for example:
+let section1 = ORKConsentSection(type: .dataGathering)
+document.sections = [section1, ...]
+
+// Add the document to a visual consent step and/or a review step:
+let step = ORKVisualConsentStep(identifier: "VisualConsentStep", document: document)
+
+// Create and present a task including this step.
+```
+
+Visual step is presented as:
+
+<kbd><img src="VisualStep_Images/VisualStep_1.png" width="500"></kbd>
+
+> Consent overview screen (ORKConsentSectionTypeOverview object)
+
+<kbd><img src="VisualStep_Images/VisualStep_2.png" width="500"></kbd>
+    
+> Data gathering (ORKConsentSectionTypeDataGathering object)
+
+<kbd><img src="VisualStep_Images/VisualStep_3.png" width="500"><kbd>
+    
+> Privacy (ORKConsentSectionTypePrivacy object)
+
+<kbd><img src="VisualStep_Images/VisualStep_4.png" width="500"></kdb>
+
+> Data use disclosure (ORKConsentSectionTypeDataUse object)
+
+<kbd><img src="VisualStep_Images/VisualStep_5.png" width="500">
+    
+> Time commitment (ORKConsentSectionTypeTimeCommitment object)
+
+<kbd><img src="VisualStep_Images/VisualStep_6.png" width="500"></kbd>
+
+> Type of study survey (ORKConsentSectionTypeStudySurvey object)</p>
+
+<kbd><img src="VisualStep_Images/VisualStep_7.png" width="500"></kbd>
+
+> Study tasks (ORKConsentSectionTypeStudyTasks object)
+
+<kbd><img src="VisualStep_Images/VisualStep_8.png" width="500"></kbd> 
+ 
+> Consent withdrawal (ORKConsentSectionTypeWithdrawing object)
+
+<kbd><img src="VisualStep_Images/VisualStep_9.png" width="500"></kbd>
+
+> Custom consent section (ORKConsentSectionTypeCustom object)</p>
+
+### Add a Review Step
+
+Users review the consent review document in the consent review step (`ORKConsentReviewStep`). Depending on your signature requirements, users can also be asked to enter their name and write a signature on the screen.
```

Comment:
switching from plural to gender-neutral singular in last clause--suggest changing 'users' to 'a user'

also, not sure of the usage elsewhere in ResearchKit documentation, but it might be worthwhile to give consideration to referring to those joining studies as 'participants' rather than 'users'--terminology can and does influence how developers and researchers will view their role in the process and thus how the studies and their apps get designed and built.